codalab/lib/bundle_cli.py

```diff
@@ -3222,6 +3222,9 @@ def display_blocks(self, client, worksheet_info, interpreted_blocks):
                     )
             elif mode == BlockModes.placeholder_block:
                 print('[Placeholder]', block['directive'], file=self.stdout)
+            elif mode == BlockModes.schema_block:
+                # print('[SchemaBlock]', file=self.stdout)
```

Comment:
Should we keep this?
```suggestion
                print('[SchemaBlock]', file=self.stdout)
```

Reply:
yeh, need to look into what's causing the test error first